scripts/get-ethbridge-ceth.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+# the script get the ceth amount of ethbridge module account.
+sifnodecli  q account sif1l3dftf499u4gvdeuuzdl2pgv4f0xdtnuuwlzp8 | jq '. | {coins: .value.coins} ' | jq '.[] | map(select(.denom == "ceth"))' | jq '.[]'
```

Comment:
is this the ethbridge module account in betanet?

Reply:
If not, we need a script that can get the ceth account balance of the cosmos module on sifchain that holds the ceth from lock and burn fees.